src/Core/Entities/EmergencyAccess.cs

```diff
@@ -10,8 +12,8 @@ public class EmergencyAccess : ITableObject<Guid>
     public Guid GrantorId { get; set; }
     public Guid? GranteeId { get; set; }
     [MaxLength(256)]
-    public string Email { get; set; }
-    public string KeyEncrypted { get; set; }
+    public string? Email { get; set; }
+    public string? KeyEncrypted { get; set; }
     public EmergencyAccessType Type { get; set; }
     public EmergencyAccessStatusType Status { get; set; }
     public int WaitTimeDays { get; set; }
```

Comment:
this is nullable.

Reply:
Yes but it's not a reference type so making that nullable is an actual code change and would likely require other changes. I did my best to only let this be annotations and not a change in current behavior.

Reply:
I see